code/development-env/docker/docker-compose-app.yml

```diff
@@ -13,7 +13,7 @@ x-common-variables: &common
 services:
   datalab-auth:
     image: node:8
-    working_dir: /usr/src/app
+    working_dir: /usr/src/app/workspaces/auth-service
```

Comment:
Oh yeah, sorry forgot to change these as part of the refactor. Thanks for picking them up,